src/desktop/apps/experimental_auth_modal/templates/meta.jade

```diff
@@ -0,0 +1,8 @@
+- var title = 'Art Basel and more on Artsy | Miami Week 2017'
+- var description = 'Artsy is your guide to Miami Week 2017. Explore the fairs, editorial coverage, Artsy events, and more—all in one place. December 4-10.'
+
+title= title
+meta( property="og:title", content=title )
+meta( name="description", content=description )
+meta( name="og:description", content=description )
+meta( property="twitter:description", content=description )
```

Comment:
@xtina-starr - Now that we're using React 16 could you try something for me? Rather than passing in a `meta.jade` template can you try to pass in a react component that is wrapped in a fragment? Stitch was built to handle any kind of template type, so i think this should work...  The way we can slowly phase out jade completely and rely entirely on react. Something like:

```javascript
// Head.tsx 

import React, { Fragment, SFC } from 'react'

interface {
  title: string
  description: string
}

export const Meta: SFC<Props> = (props) => {
  return (
     <Fragment>
       <meta property="og:title" content={props.title} />
       ... 
     </Fragment>
  )
}
```

And then later in the stitch configuration something like 

```javascript
blocks: {
  head: Head,
  ... 
}